general/virtuoso-features-matrix.ttl

```diff
@@ -0,0 +1,237 @@
+@prefix ns29:	<http://data.openlinksw.com/oplweb/product_feature/WebServiceDeploymentPlatform#> .
+@prefix ns30:	<http://data.openlinksw.com/oplweb/product_feature/WebServicestoRDFLinkedData#> .
+@prefix dt:	<http://www.w3.org/2001/XMLSchema#> .
+@prefix ns32:	<http://data.openlinksw.com/oplweb/product_feature/AttributeBasedAccessControls#> .
+@prefix ns33:	<http://data.openlinksw.com/oplweb/product_release/VirtuosoRelease8x#> .
+@prefix ns34:	<http://data.openlinksw.com/oplweb/product_feature/CustomInferenceRules#> .
+@prefix ns35:	<http://data.openlinksw.com/oplweb/product_feature/EnhancedRelationMapping#> .
+@prefix ns36:	<http://data.openlinksw.com/oplweb/product_feature/column-wise-data-storage#> .
+@prefix ns37:	<http://data.openlinksw.com/oplweb/product_release/VirtuosoRelease70x#> .
+@prefix ns38:	<http://data.openlinksw.com/oplweb/product_feature/data-compression#> .
+@prefix ns39:	<http://data.openlinksw.com/oplweb/product_feature/elastic-cluster#> .
+@prefix ns40:	<http://data.openlinksw.com/oplweb/product_feature/new-extractor-cartridges#> .
+@prefix ns41:	<http://data.openlinksw.com/oplweb/product_feature/new-meta-cartridges#> .
+@prefix ns42:	<http://data.openlinksw.com/oplweb/product_release/VirtuosoReleaseRowStoreClusterCloud64x#> .
+@prefix ns43:	<http://data.openlinksw.com/oplweb/product_release/VirtuosoReleaseRowStoreSingle64x#> .
+@prefix ns44:	<http://data.openlinksw.com/oplweb/product_feature/vectored-execution#> .
+@prefix ns45:	<http://data.openlinksw.com/oplweb/product_feature/LinkedDataPublishing#> .
+@prefix ns46:	<http://data.openlinksw.com/oplweb/product_feature/virtuoso-server-high-performance#> .
+@prefix ns47:	<http://www.openlinksw.com/temp#> .
+@prefix ns48:	<urn:website:new:rules#> .
+@prefix ns49:	<urn:webdev:virtuoso#> .
+
+ns1:this
+	schema:name	"Clustering for Fault Tolerance Load Balancing" , "Clustering for Fault Tolerance and/or Load Balancing" ;
+	feat:isFeatureOf ns3:this , ns4:this ;
+	rdf:type	feat:ProductFeature .
+ns6:this
+	schema:name	"Configurable Policy based Security" , "Configurable Policy-based Security" ;
+	feat:isFeatureOf	ns3:this , ns4:this , ns7:this ;
+	rdf:type	feat:ProductFeature .
+ns8:this
+	schema:name	"Content Negotiation Support including RDF handling" ;
+	feat:isFeatureOf	ns3:this , ns4:this , ns7:this ;
+	rdf:type	feat:ProductFeature .
+ns9:this
+	schema:name	"Distributed Transactions Support 2 Phase Commit" , "Two-Phase Commit (2PC) in Distributed Transactions" ;
+	feat:isFeatureOf	ns3:this , ns4:this , ns7:this ;
+	rdf:type	feat:ProductFeature .
+ns10:this
+	schema:name	"HTTP Compliant Application Server" , "HTTP-Compliant Application Server" ;
+	feat:isFeatureOf	ns3:this , ns4:this , ns7:this ;
+	rdf:type	feat:ProductFeature .
+ns11:this
+	schema:name	"Industry Standard Query Language Support SPARQL" , "Industry Standard SPARQL Query Language Support" ;
+	feat:isFeatureOf	ns3:this , ns4:this , ns7:this ;
+	rdf:type	feat:ProductFeature .
```

Comment:
Why do these (and many others below) have two `schema:name` values? In most (probably all) cases, one is better than the other, in varying ways (better grammar, better pluralization, better punctuation, etc.)...

Also, note that the mix of tabs and spaces makes human editing more challenging. I suggest always using spaces for indents, which many tools can be set to do even when you're typing tabs.

Reply:
These issues stem from the original output of this document before I made additions. Currently evaluating solutions.